spec/requests/tricky_operations_spec.rb

```diff
@@ -57,6 +57,12 @@
       before { disallow_operation('create_resource', source_class: Comment, related_records_with_context: related_records_with_context) }
 
       it { is_expected.to be_forbidden }
+
+      context 'which is out of scope' do
+        let(:policy_scope) { Article.none }
+
+        it { is_expected.to be_forbidden }
```

Comment:
@valscion I just realized while testing that instead of being forbidden this should probably be a not found response. I'm going to rework the processor a bit.

Reply:
Super :relaxed:. Take your time